### PR TITLE
 Fixed the iframe aspect ratio styles to avoid conflicts

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Fixed the aspect ratio selectors

# Need Review By (Date)
- asap

# Urgency
- critical

# Steps to Test
1. checkout this branch
1. create some cards/text areas etc with youtube videos.
1. ensure they aren't cut off and have a 16:9 ratio.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: jumpstart_ui.module [hunk 1]
```diff
@@ -91,3 +91,11 @@ function jumpstart_ui_preprocess_layout(&$variables) {
     $variables['layout_builder_admin'] = TRUE;
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function jumpstart_ui_preprocess_media(&$variables) {
+  $variables['attributes']['class'][] = 'media-entity-wrapper';
+  $variables['attributes']['class'][] = $variables['elements']['#media']->bundle();
```
Comment (sherakama) [inline jumpstart_ui.module:100]: Were there really not enough classes already? _shrugs_

**Review by sherakama — APPROVED** [review]
Seems fine. GTG

File: jumpstart_ui.module (hunk 1 above)
Comment (pookmish) [reply jumpstart_ui.module:100]: that element actually has 0 classes when logged out. when logged in it only has the `contextual-container` or whatever it is.

**Comment by jenbreese** [comment]
You are fast. i just sat back down.

**Pull request merged**